### PR TITLE
Test with PHP-Standard-Library

### PR DESCRIPTION
**Pull request by azjezz**

### CODE REVIEW

File: bin/test-with-real-projects.sh [hunk 1]
```diff
@@ -30,13 +30,20 @@ proxymanager)
 	~/project/psalm --monochrome
 	;;
 
+psl)
+	git clone git@github.com:azjezz/psl.git
+	cd psl
+	composer install --ignore-platform-reqs
```
Comment (azjezz) [inline bin/test-with-real-projects.sh:36]: ext-bcmath is required, but it is missing from CI.

Comment (azjezz) [reply bin/test-with-real-projects.sh:36]: @muglug build seems to pass as psalm uses the stubs for bcmath, do you think this is fine or should i add ext-bcmath to the CI?

Comment (muglug) [reply bin/test-with-real-projects.sh:36]: Tests passed, so I'm happy

**Comment by muglug** [comment]
Thanks!

**Pull request merged**